faunadb/client.py

```diff
@@ -12,6 +12,34 @@
 from faunadb.request_result import RequestResult
 from faunadb._json import parse_json_or_none, to_json
 
+class _LastTxnTime(object):
+  """Wraps tracking the last transaction time supplied from the database."""
+  def __init__(self):
+    self.lock = threading.Lock()
```

Comment:
This attributes should have a `_` prefix to "make" them private. That way you can have more idiomatic getters with the `@property`  decorator.